docs/source/locale/uk/LC_MESSAGES/tutorial.po

```diff
@@ -101,6 +101,19 @@ msgstr "Організатор може редагувати аукціон ті
 msgid "When this period ends 403 error will be returned on editing attempt:"
 msgstr "Після закінчення цього періоду на спробу редагування буде повернуто помилку 403: "
 
+msgid "Organizer can set *enquiryPeriod.endDate*. The difference between the date and *tenderPeriod.endDate* should not be less than 5 working days."
+msgstr "Організатор має можливість встановити *enquiryPeriod.endDate*. Ця дата повинна бути не пізніша, ніж за 5 днів до *tenderPeriod.endDate*."
+
+msgid "If *enquiryPeriod.endDate* provided by Organizer is closer to *tenderPeriod.endDate* than 5 days "
```

Comment:
"......closer to *tenderPeriod.endDate* than 5 days......" sounds a little bit unnatural